src/contexts/UrlContextProvider.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines */
```

Comment:
this is fine given we will refactor UrlContextProvider in #286 . in this PR, can we extend the setting value instead?
```suggestion
/* eslint max-lines: ["error", 350] */
```